dao/SeriesDao.js

```diff
@@ -60,47 +65,55 @@ module.exports.readByName = async (seriesName) => {
     if (!result) {
         throw new NotMatchedError();
     }
-    return result;
+    return new SeriesDTO(result);
 };
 
 /**
  * 계열 전체 조회
  *
- * @param {number} pagingIndex
- * @param {number} pagingSize
- * @returns {Promise<Series[]>}
+ * @param {PagingVO} pagingVO
+ * @returns {Promise<ListAndCount<SeriesDTO>>} listAndCount
  */
-module.exports.readAll = (pagingIndex, pagingSize) => {
+module.exports.readAll = ({ pagingIndex, pagingSize, order }) => {
```

Comment:
💭 이건 중요한건 아니지만, 다른 파일에서는  아래처럼 작성하셨는데 가독성이 좀 더 좋아보이고 코드 통일성면에서도 더 좋을것 같아서 남깁니다! 동작하는데는 전혀 문제 없으니 쓰루하셔도 됩니다 :)
 
```suggestion
module.exports.readAll = (pagingVO) => {
    const { pagingSize, pagingIndex, order } = pagingVO;
```

Reply:
오 꼼꼼한 리뷰 감사합니다!!!
({ pagingIndex, pagingSize, order })으로 통일 하겠습니다!